test/fixtures/m3u8/master-fmp4.js

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
This file and its expected parse is new. It's cribbed from Apple's advanced bipbop example with fragmented mp4s.